src/components/MissionProfile/MissionProfile.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import '../../pages/MyProfile/MyProfile.css';
+
+const MissionProfile = () => {
+  const missions = useSelector((state) => state.missions.missions);
+  const joinedMissions = missions.filter((mission) => mission.reserved);
```

Comment:
👍 You're on the right track with using Redux for state management! However, there's an opportunity to further enhance your code's structure:

- [ ] [Optional] Instead of handling the filter logic within your DOM component, I highly recommend moving this filtering functionality to the appropriate Redux slice. By doing so, you can centralize your state management in Redux and import the filtered data into your component. This approach adheres to the core concept of Redux, where state management is centralized, making it easier to maintain and reuse state logic across your application.

By making this adjustment, you'll have a cleaner separation of concerns and can efficiently manage state functionality within Redux. Keep up the great work! 👍🚀